rules.d/60-persistent-storage.rules.in

```diff
@@ -161,6 +161,17 @@ ENV{ID_PART_ENTRY_SCHEME}=="gpt", ENV{ID_PART_ENTRY_NAME}=="?*", SYMLINK+="disk/
 # the right device, to access specific disks in a race-free fashion)
 ENV{DISKSEQ}=="?*", ENV{ID_IGNORE_DISKSEQ}!="1", SYMLINK+="disk/by-diskseq/$env{DISKSEQ}$env{.PART_SUFFIX}"
 
+# by-parttype links (Discoverable Partitions Spec)
+ENV{ID_PART_ENTRY_SCHEME}=="gpt", ENV{ID_BOOT_DISK}=="1", ENV{ID_FS_USAGE}=="filesystem|other|crypto", GOTO="persistent_storage_by_parttype_start"
```

Comment:
why check ID_FS_USAGE?

Reply:
maybe better keep the check, i think it makes sense to stay out of raid stuff, lvm is weird.